app/components/user-page/course-progress-list.hbs

```diff
@@ -5,6 +5,7 @@
 
   <div class="grid grid-cols-1 gap-3 lg:grid-cols-2">
     {{#each this.courseParticipationGroups as |courseParticipationGroup|}}
+      {{! @glint-ignore }}
```

Comment:
@ryan-gang let's do `@glint-expect-error` instead of `@glint-ignore`. `@glint-ignore` won't complain if the error isn't present anymore.

More on this here: https://typed-ember.gitbook.io/glint/using-glint/directives